x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,65 @@
+.appSearchResultContainer {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
```

Comment:
[TODO for myself to investigate later] It's not clear to me at first glance why this container needs to be a flex container - I wonder if we can potentially convert it to a plain div or possibly even lose the wrapper

Reply:
Yes, that is definitely a possibility. I tried to clean these up as best I could, I managed to remove a boat load of old cruft but there are probably a couple of things here and there that are still lingering, like this.

Reply:
I cleaned this up, it renders identically
ace583e36609d87bb3499ddf5a271be41b15b209